ILRepack/RepackLogger.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace ILRepacking
 {
-    class RepackLogger : ILogger
+    public class RepackLogger : ILogger
```

Comment:
Oh, actually, I've forgot to add the non-Logger default ctor "ILRepack(RepackOptions) : this(options, new RepackLogger())", which is IMO better, as the less we expose, the better.